src/adaptors/curve/index.js

```diff
@@ -134,13 +134,14 @@ const getPoolAPR = (pool, subgraph, gauge, crvPrice) => {
     .reduce((a, b) => a.plus(b));
   const assetPrice = nominator.div(denominator);
   try {
-    const poolAPR = crvPriceBN
-      .times(inflationRate)
+    const poolAPR = inflationRate
```

Comment:
@crnch @slasher125 There was wrong rewardApy calculations, fixed it
Src: https://github.com/curvefi/curve-js/blob/c82958fbdaaee67546c6d78c5d9497eadebe974c/src/pools.ts#L281